live-examples/js-examples/intl/intl-supportedvaluesof.js

```diff
@@ -0,0 +1,14 @@
+console.log(Intl.supportedValuesOf('calendar'));
+console.log(Intl.supportedValuesOf('collation'));
+console.log(Intl.supportedValuesOf('currency'));
+console.log(Intl.supportedValuesOf('numberingSystem'));
+console.log(Intl.supportedValuesOf('timeZone'));
+console.log(Intl.supportedValuesOf('unit'));
+// expected output: Array ['key'] (for each key)
+
+try {
+  Intl.supportedValuesOf('someInvalidKey');
+} catch (err) {
+  console.log(err.toString());
```

Comment:
REVIEWER: Note that I was forced to do `toString` here to get a string - without this the output is an object. That seems like a bug in the example infrastructure